client/src/components/apiHandler.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+
+export const getApiData = (props) => {
```

Comment:
i'll ignore this file since it's not integrated

Reply:
it's called in App.js